apps/server/src/modules/h5p-editor/service/h5p-ajax-endpoint.service.ts

```diff
@@ -0,0 +1,11 @@
+import { H5PAjaxEndpoint, H5PEditor } from '@lumieducation/h5p-server';
+
+export const H5PAjaxEndpointService = {
```

Comment:
Service zu Provider umbenennen und in ein provider Odner verschieben.

Reply:
solved